README.md

```diff
@@ -95,17 +111,6 @@ NavigationUI.shared.lineColor = .yellow
 
 * `routeControllerDidCancelNavigation`: Fired when the user taps `Cancel`. You are responsible for dismissing the UI
 
-## Examples
-
-We provide examples in Swift and Objective-C. Run `carthage update --platform ios` from the root folder and open MapboxNavigation.xcodeproj to try it out.
-
-### Running the example app
-
-1. If running in the simulator, you can simulate the user location by selecting `Debug` -> `Location` -> `City Bicycle Ride`
```

Comment:
The default simulated location preset is either Apple or None, which doesn’t do a whole lot. A nod to this setting would be helpful for developers who might forget about this setting’s existence.